cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,32 @@
+import time
+from django.db import connections
+from django.db.utils import OperationalError
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    """Django command to pause execution until database is available"""
+
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database...")
+        db_conn = None
+        start_time = time.time()
+
+        while not db_conn:
+            try:
+                db_conn = connections["default"]
+            except OperationalError:
+                if time.time() - start_time > 10:
```

Comment:
Consider making the timeout period configurable by adding an option to the command. This would make the script more flexible for different environments where the database might take longer to become available.